spec/views/login_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+RSpec.feature 'Logins', type: :feature do
+  background { visit new_user_session_path }
+
+  it 'displays email field' do
+    expect(page).to have_field('user[email]')
+  end
+
+  it 'display Password field' do
+    expect(page).to have_field('user[password]')
+  end
+
+  it 'Display Login' do
+    expect(page).to have_content 'Log in'
+  end
+
+  context 'Submit a form' do
+    it 'Submit form without email and password' do
+      click_button 'Log in'
+      expect(page).to have_content 'Invalid Email or password.'
+    end
+  end
+
+  it 'Submit form with incorrect credentials' do
+    within 'form' do
+      fill_in 'Email', with: 'koome@gmail.com'
+      fill_in 'Password', with: 'kk123454321'
+    end
+    click_button 'Log in'
+    expect(page).to have_content 'Invalid Email or password.'
```

Comment:
- [ ] It will be nice to also check the content of the `it` block. Incorrect credentials should not be able to submit form.